src/Shared/ExceptionHandling.cs

```diff
@@ -301,22 +301,26 @@ internal static void DumpExceptionToFile(Exception ex)
                 if (s_dumpFileName == null)
                 {
                     Guid guid = Guid.NewGuid();
-                    string tempPath = Path.GetTempPath();
+                    string dumpDirEnv = Environment.GetEnvironmentVariable("MSBUILDDEBUGPATH");
```

Comment:
Can you just make this:

```C#
string dumpDir = Environment.GetEnvironmentVariable("MSBUILDDEBUGPATH") ?? Path.GetTempPath();
```

And get rid of the other variable?  Or am I missing something?

Reply:
Just being safe in case MSBUILDDEBUGPATH is set but empty.

Reply:
You could do this too, but calling .GetEnvironmentVariable twice seemed uglier.

```
string dumpDir = !string.IsNullOrEmpty(Environment.GetEnvironmentVariable("MSBUILDDEBUGPATH"))
                            ? Environment.GetEnvironmentVariable("MSBUILDDEBUGPATH")
                            : Path.GetTempPath();
```